bindings/nodejs/lib/wallet/account.ts

```diff
@@ -879,20 +879,20 @@ export class Account {
     }
 
     /**
-     * Prepare a send amount transaction, useful for offline signing.
+     * Prepare a send transaction, useful for offline signing.
```

Comment:
prepare to send would be better now, old phrasing was weird anyway

Reply:
Changed.